airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle

```diff
@@ -17,13 +17,6 @@ plugins {
     id "java-test-fixtures"  // https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures
 }
 
-test {
-    testLogging {
-        // TODO: Remove this after debugging
-        showStandardStreams = true
```

Comment:
I've made it so that we log everything all the time. If it's too verbose we can edit the log4j config instead.